### PR TITLE
add time to strings on results pages

### PR DESCRIPTION
**Pull request by alismx**
## Related Issue or Background Info

- #3225 

## Changes Proposed

- Move date format functions to the date utility
- Update all places where the test date is reported to display time
- Add tests for date format functions
- Additional change outside the scope of this ticket: Add git volume mounts to our cypress and docker-compose files
  - This resolves an issue where the frontend is not setting REACT_APP_CURRENT_COMMIT when starting in a docker container
  - This resolves an issue with a failing Cypress health check test.

## Additional Information

- Talked with Tim about where to add time, and we decided that we should add it to any place that displays the test date-time; if I missed any spots, lmk

## Screenshots / Demos
deepin-screen-recorder_Select area_20220201135235

### Infrastructure
- [x] **Consult the results of the `terraform-plan` job inside the "Terraform Checks" workflow run for this PR. Confirm that there are no unexpected changes!**

### Design
- [x] Any UI/UX changes have a designer as a reviewer, and changes have been approved
- [x] Any large-scale changes have been deployed to `test`, `dev`, or `pentest` and smoke-tested by both the engineering and design teams

### Content
- [x] Any content changes (including new error messages) have been approved by content team

### Support
- [x] Any changes that might generate new support requests have been flagged to the support team
- [x] Any changes to support infrastructure have been demo'd to support team

### Testing
- [x] Includes a summary of what a code reviewer should verify

### Changes are Backwards Compatible
- [x] Database changes are submitted as a separate PR
  - [x] Any new tables that do not contain PII are accompanied by a GRANT SELECT to the no-PHI user
  - [x] Any changes to tables that have custom no-PHI views are accompanied by changes to those views
        (including re-granting permission to the no-PHI user if need be)
  - [x] Liquibase rollback has been tested locally using `./gradlew liquibaseRollbackSQL` or `liquibaseRollback`
  - [x] Each new changeset has a corresponding [tag](https://docs.liquibase.com/change-types/community/tag-database.html)
- [x] GraphQL schema changes are backward compatible with older version of the front-end

### Security
- [x] Changes with security implications have been approved by a security engineer (changes to  authentication, encryption, handling of PII, etc.)
- [x] Any dependencies introduced have been vetted and discussed

### CODE REVIEW

File: frontend/cypress/docker-compose.yml [hunk 1]
```diff
@@ -63,6 +63,7 @@ services:
       REACT_APP_OKTA_URL: http://cypress:8088
     volumes:
       - ../:/frontend/
+      - ../../.git/:/frontend/.git
```
Comment (nickclyde) [inline frontend/cypress/docker-compose.yml:66]: Does this help with running the health check e2e test in the compose setup?

**Review by nickclyde — APPROVED** [review]
Looks great!

File: frontend/cypress/docker-compose.yml (hunk 1 above)
Comment (alismx) [reply frontend/cypress/docker-compose.yml:66]: Yes. This resolves an issue where the frontend is not setting REACT_APP_CURRENT_COMMIT when starting in a docker container and resolves an issue with a failing Cypress health check test.

It should have been included in my last cypress PR, but seems to have been pulled out.

**Review by nathancrtr — COMMENTED** [review]
These changes look good, but I was a little surprised to see that we didn't have existing snapshot tests for these components. Could you consider adding them for the `TestResultPrintModal` and `TestResultDetailsModal` components?

Fortunately it's very simple to do so: you can simply take the `container` HTMLElement from the result returned by the `render()` call, and invoke the snapshot test with:

```
expect(container).toMatchSnapshot();
```

Example:

https://github.com/CDCgov/prime-simplereport/blob/f78774a1d8692a990eaff683d8e957e4444b6515/frontend/src/app/patients/EditPatient.test.tsx#L221-L246

**Comment by alismx** [comment]
@vz3 Could I get design approval for the timestamp addition? It looks like I've got some UI review pending.

**Review by nathancrtr — APPROVED** [review]
LGTM!

**Review by vz3 — APPROVED** [review]
Looks good here

**Review by zdeveloper — APPROVED** [review]
LGTM! thanks for getting this done!

**Pull request merged**